sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadSupport.scala

```diff
@@ -156,9 +189,10 @@ object ParquetReadSupport {
   def clipParquetSchema(
       parquetSchema: MessageType,
       catalystSchema: StructType,
+      useFieldId: Boolean,
```

Comment:
Is it possible to not change the order of the fields and add `useFieldId` at the end of the list of arguments instead?

Reply:
Sure

Reply:
I suppose this still needs to be updated...